tests/scheme.ts

```diff
@@ -87,7 +88,7 @@ switch (Scheme) {
     CredentialBuilder = BBSCredentialBuilder;
     Credential = BBSCredential;
     encodeMessageForSigningIfPS = (msg) => msg;
```

Comment:
You don't have to define this as it same as previously defined.

Reply:
It was declared only with a type definition without the actual implementation.